### PR TITLE
Suggest colors for syntax

### PR DESCRIPTION
**Pull request by Ozyugoo**

### CODE REVIEW

File: practice/004/ex.ml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+open OUnit2
+
+module type Testable = sig
+  val length : 'a list -> int
+end
+
+module Make(Tested: Testable) : sig val v : test end = struct
+  let v = "length" >::: [
+    "nil" >:: (fun _ -> assert_equal (Tested.length []));
+    "cons" >:: (fun _ -> assert_equal (Tested.length ["a"; "b"; "c"]));
```
Comment (cuihtlauac) [inline practice/004/ex.ml:11]: You must specify the expected return value in both cases here

**Review by cuihtlauac — CHANGES_REQUESTED** [review]
Thanks @Ozyugoo. Yes this is going in the right direction. Make sure compilation is successful before pushing a PR. Also you should and newlines characters at end of file, when missing

**Comment by cuihtlauac** [comment]
You have mixed two PRs in this one. Work on PR #2168 must be made in a separate PR

File: practice/004/ex.ml (hunk 1 above)
Comment (Ozyugoo) [reply practice/004/ex.ml:11]: Thank you, on it.

**Comment by Ozyugoo** [comment]
> You have mixed two PRs in this one. Work on PR #2168 must be made in a separate PR

I will do that, thank you

**Pull request closed without merging**